web/template/change-password.gotmpl

```diff
@@ -11,22 +11,32 @@
         {{ template "success-banner" "You have successfully changed your password." }}
       {{ end }}
 
-      <h1 class="govuk-heading-xl">Change password</h1>
-
+      <fieldset class="govuk-fieldset">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+          <h1 class="govuk-fieldset__heading">Change your password</h1>
+      </legend>
+
+      <p class="govuk-body">Your password needs to have:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>at least 8 characters</li>
+        <li>at least one lower-case and one capital letter</li>
+        <li>at least one number</li>
+      </ul>
+      <p class="govuk-body">Never share your password with anyone.</p>
+      
       <form class="form" action="{{ prefix "/change-password" }}" method="post">
         <div class="govuk-form-group">
-          <label class="govuk-label govuk-label--m" for="f-currentpassword">
+          <label class="govuk-label" for="f-currentpassword">
             Current password
           </label>
 
           <input class="govuk-input" id="f-currentpassword" name="currentpassword" type="password">
         </div>
 
         <fieldset class="govuk-fieldset">
```

Comment:
Because you've opened `<fieldset>` on line 14, this one should be removed now [1]